compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/fuse_logical_objectfifo_into_workgroup.mlir

```diff
@@ -0,0 +1,35 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(iree-amdaie-fuse-logicalobjectfifo-into-workgroup)" %s | FileCheck %s
+
+// CHECK-LABEL: @fuse_logical_objectfifo_into_workgroup
+// CHECK:     %[[ALLOC0:.*]] = memref.alloc() : memref<32x1024xi32>
+// CHECK:     %[[ALLOC1:.*]] = memref.alloc() : memref<32x64xi32, 1>
+// CHECK-NOT: amdaie.logicalobjectfifo.from_memref
+// CHECK:     scf.forall
+// CHECK:       amdaie.workgroup
+// CHECK:         %[[FROMMEMREF1:.*]] = amdaie.logicalobjectfifo.from_memref %[[ALLOC1]]
+// CHECK-SAME:      memref<32x64xi32, 1> -> !amdaie.logicalobjectfifo<memref<32x64xi32, 1>>
+// CHECK:         %[[FROMMEMREF0:.*]] = amdaie.logicalobjectfifo.from_memref %[[ALLOC0]]
+// CHECK-SAME:      memref<32x1024xi32> -> !amdaie.logicalobjectfifo<memref<32x1024xi32>>
+// CHECK:         %[[DMA0:.*]] = amdaie.dma_cpy_nd
+// CHECK-SAME:      %[[FROMMEMREF1]]
+// CHECK-SAME:      %[[FROMMEMREF0]]
+// CHECK-SAME:      (!amdaie.logicalobjectfifo<memref<32x64xi32, 1>>, !amdaie.logicalobjectfifo<memref<32x1024xi32>>)
+module {
```

Comment:
Maybe good to add in a following PR... It's good to add another test for multiple workgroups case.